other/require_pdb/require_pdb.yaml

```diff
@@ -17,26 +17,29 @@ spec:
   validationFailureAction: audit
   background: false
   rules:
-  - name: require-pdb
-    match:
-      resources:
-        kinds:
-        - Deployment
-    preconditions:
-      any:
-      - key: "{{request.operation || 'BACKGROUND'}}"
-        operator: Equals
-        value: CREATE
-    context:
-    - name: pdb_count
-      apiCall:
-        urlPath: "/apis/policy/v1/namespaces/{{request.namespace}}/poddisruptionbudgets"
-        jmesPath: "items[?label_match(spec.selector.matchLabels, `{{request.object.spec.template.metadata.labels}}`)] | length(@)"
-    validate:
-      message: "There is no corresponding PodDisruptionBudget found for this Deployment."
-      deny:
-        conditions:
-          any:
-          - key: "{{pdb_count}}"
-            operator: LessThan
-            value: 1
\ No newline at end of file
+    - name: require-pdb
+      match:
```

Comment:
Since you're updating this, go ahead and add `any`/`all` under the `match` object here to bring it current.

Reply:
done